packages/formation/sass/utilities/_flexbox.scss

```diff
@@ -2,11 +2,11 @@
 Flex shorthand
 =========================*/
 $flex-values: (
-  1 0 0,
-  2 0 0,
-  3 0 0,
-  4 0 0,
-  1 0 0,
+  1 0 0%,
+  2 0 0%,
+  3 0 0%,
+  4 0 0%,
+  1 0 0%,
   0 0 auto
```

Comment:
I had to do a bit of learnin' for this. According to [the docs](https://developer.mozilla.org/en-US/docs/Web/CSS/flex), in the three-value syntax, the third value is the width for the `flex-basis`.

Additionally:

![image](https://user-images.githubusercontent.com/2008881/98037651-c3c80200-1dd0-11eb-9cdd-68f1086d1c95.png)


So, the unit (`px`, `%`, etc.) is required.

Reply:
Thanks for looking that up! It might be worth sweeping through for other instances where this could happen. You would think that a `0` is `0` no matter what the unit is, but 🤷